src/test/compile-fail/E0204.rs

```diff
@@ -10,18 +10,20 @@
 
 struct Foo {
     foo: Vec<u32>,
+    //~^ this field doesn't implement `Copy`
 }
 
 impl Copy for Foo { }
 //~^ ERROR E0204
-//~| NOTE field `foo` does not implement `Copy`
+//~| NOTE field `foo` in struct `Foo` doesn't implement `Copy`
```

Comment:
do we need to say "in struct `Foo`" here?  If we're also giving the label on the struct itself, this feels less helpful